docs/imagecustomizer/api/configuration/outputBootloader.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Configuration
+---
+
+# outputBootloader type
+
+Specifies the configuration for the output directory containing the generated bootloader components, including shim and systemd-boot.
+
+Example:
+
+```yaml
+bootloader:
+  path: /output/bootloader
+```
+
+## path [string]
+
+Required.
+
+Specifies the directory path where Prism will output the generated bootloader components.
+
+### Expected Files in the Directory
+
+After the image customization process, this directory will contain the following unsigned bootloader components:
+
+- bootx64.efi – The generated shim executable.
```

Comment:
For `arm` this will be different, right? so maybe this should be arch agnostic?

Reply:
Will revise, thanks.